src/app/users/users.component.html

```diff
@@ -1,12 +1,37 @@
-<div *ngFor="let user of users">
-  <mat-list role="list">
-    <mat-list-item role="listitem">Id: {{ user.id }}</mat-list-item>
-    <mat-list-item role="listitem">User: {{ user.name }}</mat-list-item>
-    <mat-list-item role="listitem">Email: {{ user.email }}</mat-list-item>
-    <mat-list-item role="listitem">Password: {{ user.password }}</mat-list-item>
-    <section>
-      <button mat-raised-button color="primary">Edit User</button>
-      <button mat-raised-button color="warn">Delete User</button>
-    </section>
-  </mat-list>
-</div>
+<br />
+<button mat-raised-button color="accent" routerLink="new">Create User</button>
+<br />
```

Comment:
Don't use <br />  tag. 
use css instead and clases

Reply:
done thanks!